rainbows.rb

```diff
@@ -27,3 +27,32 @@ def mix_colors(color1,color2)
 
 new_color = mix_colors(red,green)
 puts "#{color_lookup[red]} #{red} + #{color_lookup[green]} #{green} = #{color_lookup[new_color]} #{new_color}"
+
+rainbow = [
+  {:red => {r: 255,
+    g: 0,
+    b: 0}},
+  {:orange => {r: 255,
+    g: 127,
+    b: 0}},
+  {:yellow => {r: 255,
+    g: 255,
+    b: 0}},
+  {:green => {r: 0,
+    g: 255,
+    b: 0}},
+  {:blue => {r: 0,
+    g: 0,
+    b: 255}},
+  {:indigo => {r: 75,
+    b: 0,
+    g: 130}},
+  {:violet => {r: 143,
+    b: 0,
+    g:255}}
+]
+ 
+
+  rainbow.each do |i|
+    puts i.keys
+  end
```

Comment:
This is good, except that it won't allow us to do anything with each key individually (if we wanted to colorize) so doesn't necessarily plan for enhancements. But it works, which is great!